<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1" colspan="4">
     <widget class="QGroupBox" name="tradeGroup">
      <property name="title">
       <string>Trading cycle</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QDateEdit" name="tradeFrom">
         <property name="date">
          <date>
           <year>2009</year>
           <month>9</month>
           <day>25</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy.MM.dd</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Coefficients</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <widget class="QLineEdit" name="tradeW2"/>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="tradeW3"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>W2=</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="tradeW1"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>W1=</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>W3=</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>W4=</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="tradeW4"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="start_trade">
         <property name="text">
          <string>Start trade</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDateEdit" name="tradeTo">
         <property name="date">
          <date>
           <year>2009</year>
           <month>10</month>
           <day>25</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy.MM.dd</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Profit:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="tradeProfit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="start_all">
      <property name="text">
       <string>Learn and trade</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="select_file">
      <property name="text">
       <string>Select file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="select_file_label">
      <property name="text">
       <string>Selected file:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QGroupBox" name="trainGroup">
      <property name="title">
       <string>Learning cycle</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QDateEdit" name="trainFrom">
         <property name="date">
          <date>
           <year>2009</year>
           <month>7</month>
           <day>25</day>
          </date>
         </property>
         <property name="maximumTime">
          <time>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
          </time>
         </property>
         <property name="displayFormat">
          <string>yyyy.MM.dd</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Coefficients</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>W1=</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>W3=</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>W4=</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>W2=</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="trainW1"/>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="trainW4"/>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="trainW3"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="trainW2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="start_learn">
         <property name="text">
          <string>Start learn</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDateEdit" name="trainTo">
         <property name="date">
          <date>
           <year>2009</year>
           <month>9</month>
           <day>25</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy.MM.dd</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Profit:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="trainProfit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
